org.shotcut.Shotcut.yaml

```
@@ -58,7 +58,7 @@ modules:
       - '/lib/pkgconfig'
     post-install:
       - install -d /app/bin
-      - install -p -c bin/main /app/bin/whisper.cpp-main
+      - install -p -c bin/whisper-cli /app/bin
```

```suggestion
      - install -Dm755 bin/whisper-cli  -t /app/bin
```

This is probably better. And you can remove the `install -d` line 60.
`-c` is a no-op and `-p` is irrelevant as the date/times are always reset.